### ISSUE TITLE
AutoComplete | clear input field if showClear="true" 

### ISSUE THREAD

**Issue by cetincakiroglu** [primefaces/primeng#15078]
- Input fields should be cleared on `backspace` keydown if `showClear = "true"`.
